README.md

```diff
@@ -30,6 +30,9 @@ Formatters to take a JSON array and write to the stdout. Current formatters incl
 
 A custom http client for the Twilio helper library to allow us to log API requests as well as modify the User-Agent header.
 
+#### Environment variables
```

Comment:
Rename `#### Environment variables` to `### Usage with proxy`

Reply:
This is now changed